solaris/packer/solaris10.json

```diff
@@ -170,12 +163,16 @@
   ],
   "provisioners": [
     {
-      "execute_command": "chmod +x {{ .Path }}; export {{.Vars}} && cat {{.Path}} | pfexec su ",
+      "execute_command": "chmod +x {{ .Path }}; export {{.Vars}} \u0026\u0026 cat {{.Path}} | pfexec su ",
```

Comment:
Is this intended?

Reply:
Yes, it is. Using `&&` doesn't works as would be expected.